File: SpringMicroservices/microservice-music/src/main/java/com/microservice/music/infrastructure/adapters/in/rest/dto/SongDto.java
package com.microservice.music.infrastructure.adapters.in.rest.dto;

import lombok.*;

@Data
@NoArgsConstructor
@AllArgsConstructor
@Builder
public class SongDto {
    private Long idSong;
    private String title;
    private String audioUrl;
    private String lyricsEs;
    private String lyricsEn;
    private Long idAlbum;
    private Integer durationInSeconds;
    private Boolean visible;
}

File: ProyFronted/app/src/main/java/com/example/proyfronted/backend/Music/Dto/SongDto.kt
package com.example.proyfronted.backend.Music.Dto

import android.os.Parcelable
import kotlinx.parcelize.Parcelize

@Parcelize
data class SongDto(
    val idSong: Long,
    val title: String,
    val audioUrl: String,
    val lyricsEs: String?,
    val lyricsEn: String?,
    val idAlbum: Long?,
    val durationInSeconds: Int,
    val visible: Boolean = true
) : Parcelable
